.github/workflows/changeset.yaml

```
@@ -26,16 +28,22 @@ jobs:
           commit: "version packages"
           version: yarn ci:version
           publish: yarn ci:publish
+  notify-slack:
+    runs-on: ubuntu-latest
+    needs: version
+    env:
+      PUBLISHED: ${{ needs.version.outputs.hasPublished }}
+    steps:
       - name: Publish successful
-        if: steps.changesets.outputs.published == 'true'
+        if: ${{ env.PUBLISHED }}
```

```suggestion
        if: ${{ env.PUBLISHED == 'true' }}
```
In case it's some strange truthy value?

---

Yeah fair

---

Weird Github is weird